docs/mac_development.md

```diff
@@ -63,6 +69,7 @@ You can build an image with your changes and use that image in docker-compose
 2. export EDA_IMAGE=localhost/myserver:latest
 3. docker-compose -f ./tools/docker/docker-compose-mac.yml up
 
+You can now access the UI at <https://localhost:8443/eda/> with default login username and password(admin/testpass).
```

Comment:
Would you mind to add the same line for linux as well?

Reply:
sure, added